src/pages/create-program/components/create-program-settings/create-program-settings.js

```diff
@@ -235,9 +253,13 @@ class CreateProgramSettings extends React.Component {
                   content={t("create-program-page.settings.hints.success-fee")}
                   className="create-program-settings__fee-hint"
                   vertical={"bottom"}
-                  tooltipContent={t(
-                    "create-program-page.settings.hints.success-fee-description"
-                  )}
+                  tooltipContent={
```

Comment:
Do not complicate. Use {{variable}} in the translations. ("disclaimer": "After clicking the \"Confirm\" button, the invested GVT will be immediately converted to {{currency}}.")